### PR TITLE
Unnecessary double quotes in site components

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Replaced unnecessary double quotes in site/components
- Removed one or two whitespaces at EOLs (sorry, couldn't resist)
- Inserted a line before if statements where I stumbled upon (sorry, couldn't resist)

This PR is part of a set to try to separate some of the changes done in some of my previous batch PR's for site/components, which are still on hold (#12290, #12292, #12293, #12294).

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: components/com_finder/helpers/html/filter.php [hunk 1]
```diff
@@ -172,9 +172,9 @@ public static function slider($options = array())
 			);
 
 			// Populate the toggle button.
-			$html .= "<button class=\"btn\" type=\"button\" class=\"jform-rightbtn\" onclick=\"jQuery('[id=tax-"
-				. $bk . "]').each(function(){this.click();});\"><span class=\"icon-checkbox-partial\"></span> "
-				. JText::_('JGLOBAL_SELECTION_INVERT') . "</button><hr/>";
+			$html .= '<button class="btn" type="button" class="jform-rightbtn" onclick="jQuery(\'[id=tax-'
+				. $bk . ']\').each(function(){this.click();});"><span class="icon-checkbox-partial"></span> '
+				. JText::_('JGLOBAL_SELECTION_INVERT') . '</button><hr/>';
```
Comment (frankmayer) [inline components/com_finder/helpers/html/filter.php]: Of, course, I'll check it in a bit.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 257827f48b70c2dc856a24ef12f8e72834d190de<br /><br />Code review. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13183">issues.joomla.org/tracker/joomla-cms/13183</a>.</sub>

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on 257827f48b70c2dc856a24ef12f8e72834d190de<br /><br />Code review. Good!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13183">issues.joomla.org/tracker/joomla-cms/13183</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13183">issues.joomla.org/tracker/joomla-cms/13183</a>.</sub>

**Pull request merged**